src/components/LoginForm/index.js

```diff
@@ -20,59 +21,41 @@ const FIELDS = {
 
 const LoginForm = ({ onSubmit }) => {
   const { error, status } = useStatus(login);
-  const validator = useValidation(loginValidations);
+
   const {
-    values,
-    errors,
-    handleValueChange,
+    control,
     handleSubmit,
-    handleFocus,
-    handleBlur,
-    activeFields,
-    touched,
-    formHasErrors,
-  } = useForm(
-    {
-      onSubmit,
-      validator,
-      validateOnBlur: true,
-      validateOnChange: true,
-    },
-    [onSubmit],
-  );
-
-  const inputProps = useTextInputProps(
-    handleValueChange,
-    handleFocus,
-    handleBlur,
-    values,
-    errors,
-    activeFields,
-    touched,
-  );
+    formState: { errors, isDirty, isValid },
+  } = useForm();
 
   return (
     <>
-      <Input
+      <TextInput
+        name={FIELDS.email}
         label={strings.SIGN_IN.email}
+        control={control}
+        rules={validations.email}
+        errors={errors}
         keyboardType="email-address"
         autoCapitalize="none"
         testID="email-input"
-        {...inputProps(FIELDS.email)}
       />
```

Comment:
If all the accessibility stuff was added it would be nice to also add the functionality to go to the next input on return key pressed. This is present in the FormExample component of the accessibility playground repo this changes where ported from.

Reply:
Yes, this is something I'll check today, this first push was only to replace the current implementation and from this point  add new features like this

Reply:
I just added the onSubmitEditing function